inc/class-upgrade.php

```diff
@@ -42,6 +42,13 @@ public function __construct() {
 			$this->upgrade_22();
 		}
 
+		/**
+		 * Filter: 'wpseo_run_upgrade' - Runs the upgrade hook which are dependent on Wordpress SEO by Yoast
+		 *
+		 * @api string - The current version of Wordpress SEO by Yoast
```

Comment:
I can confirm this is the case in most other places. I don't know the rationale behind it. Might be something to address in a different context.